hoomd/md/pytest/test_custom_force.py

```diff
@@ -378,3 +378,37 @@ def test_torques_update(local_force_names, two_particle_snapshot_factory,
         if sim.device.communicator.rank == 0:
             assert np.count_nonzero(snap.particles.orientation
                                     - initial_orientations)
+
+
+def test_force_zeroing(force_simulation_factory, two_particle_snapshot_factory):
```

Comment:
How does this test make sure the forces are zeroed every timestep before calling `set_forces`? It doesn't seem to me that its doing that.

Reply:
I'm not seeing it either. You assign the whole array to the value `timestep`. Maybe you mean to use `+= timestep`?

Reply:
That is what I was thinking in my head @joaander; but apparently not in my code. Sorry @tommy-waltmann for the confusion on my part.